Release 4.11.1 (#262)
blackbaud/skyux-datetime: CHANGELOG.md

# 4.11.1 (2021-05-20)

> - Fixed the date range picker to disable the inner form controls when consumers use a reactive form control during initialization or the `disabled` input. [#261](https://github.com/blackbaud/skyux-datetime/pull/261)

Reviewer: ```suggestion
- Fixed the date range picker to disable the inner form controls when consumers use reactive form controls during initialization and when they use the `disabled` input. [#261](https://github.com/blackbaud/skyux-datetime/pull/261)
```
